libraries/GCS_MAVLink/GCS.h

```diff
@@ -805,7 +806,6 @@ class GCS_MAVLINK
     // number of extra ms to add to slow things down for the radio
     uint16_t         stream_slowdown_ms;
     // last reported radio buffer percent available
```

Comment:
This comment should be moved to the respective line on the `last_radio_status` structure

Reply:
Oops.

Reply:
I removed the orphan comment in my recent comit